scripts/package.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+set -e
+
+BASE_TAG=""
+LICENSE_FILE="./License"
+PACKAGES=()
+FINAL_TAG=""
```

Comment:
Don't you think it would be more convenient to have some default tags? What the default final tag should / could be is up to debate, but leaving it empty also seems cumbersome. You've been using `customer:v1` in your examples, but a generic name for documentation, examples and the default here could be `aica-technology/app`, potentially with the base tag added.

Also nitpick, up until now the default license file has been lowercased.

```suggestion
BASE_TAG="latest"
LICENSE_FILE="./license"
PACKAGES=()
FINAL_TAG="aica-technology/app:${BASE_TAG}"
```

Reply:
Actually we don't have a `latest` tag for `base` in the registry (or for any other images either by the way) as it makes quite hard to do licensing check with it so that's no possible and I will revert that part of the commit.

Reply:
Ah yes good point 👍